src/main/resources/assets/js/application/report/views/tdpr-directive-report.html

```diff
@@ -0,0 +1,8 @@
+<th class="slot hour-cell">
+    <a ui-sref="tdpr.interviewer.home({isRecruiter : true, id : personInfo.id, personName : personInfo.firstName + ' ' + personInfo.lastName})">
+        <span>{{personInfo.lastName + ' ' + personInfo.firstName}}</span>
+    </a>
+</th>
+<td class="slot hour-cell">{{reportData.numberOfInitMinutes/60}}</td>
```

Comment:
Business logic/calculations should not be performed in views. It would be good to perform this calculation in a controller/service method.